subsys/caf/modules/ble_smp.c

```diff
@@ -56,6 +56,7 @@ static bool event_handler(const struct event_header *eh)
 				module_set_state(MODULE_STATE_ERROR);
 			} else {
 				LOG_INF("Service initialized");
+				LOG_INF("MCUboot image version: %s", CONFIG_MCUBOOT_IMAGE_VERSION);
```

Comment:
Should we rely on Kconfig option value here? Or we should use `boot_read_bank_header` to rely on the actual header from the image?

Reply:
We may encounter problems if the image would be signed externally to our build system